tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -26,6 +26,23 @@
 )
 from tardis.montecarlo.montecarlo_numba import njit_dict
 from numba.typed import List
+import tqdm.notebook as tq
+from tqdm import tqdm
+
+packet_pbar = tq.tqdm(
+    total=860000,
```

Comment:
We can't hardcode the number of packets or iterations.  These numbers need to be retrieved from the tardis config object.